messaging-activemq/pom.xml

```diff
@@ -244,5 +244,10 @@
             <scope>test</scope>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-stomp-protocol</artifactId>
+            <version>2.6.3.jbossorg-001</version>
```

Comment:
unparametrized version does not seem correct

Reply:
Sorry this was a leftover